.github/workflows/build.yml

```diff
@@ -189,18 +189,27 @@ jobs:
         run: |
           go test -v ./internal/apex/...
 
-      - name: Build
-        id: build-apex
+      - name: Build Apexd
+        id: build-apexd
         run: |
           CGO_ENABLED=0 go build -o apexd-$(go env GOOS)-$(go env GOARCH) -v ./cmd/apexd
           echo "artifact-name=apexd-$(go env GOOS)-$(go env GOARCH)" >> $GITHUB_OUTPUT
         shell: bash
 
+      - name: Build Apexctl
+        id: build-apexctl
+        run: |
+          CGO_ENABLED=0 go build -o apexctl-$(go env GOOS)-$(go env GOARCH) -v ./cmd/apexctl
```

Comment:
This isn't a problem for this PR, but I wish we weren't duplicating `go` commands in `build.yaml` and the `Makefile`. I'd love to change these to use `make`, but we can figure that out later ...